README.md

```diff
@@ -1 +1,215 @@
+      ```bash
+      helm repo add localstack-charts https://localstack.github.io/helm-charts
+      helm install localstack localstack-charts/localstack \
+       --version 0.4.3 \
+       --set image.tag="1.2.0" \
+       --set startServices="s3" \
+       --set service.type=ClusterIP \
+       --set livenessProbe.initialDelaySeconds=25
+      kubectl wait --for=condition=ready --all pod -n fybrik-notebook-sample --timeout=120s
+      ```
+
+3. Create a port-forward to communicate with localstack server:
+      ```bash
+      kubectl port-forward svc/localstack 4566:4566 &
+      ```
+4. Use [AWS CLI](https://aws.amazon.com/cli/) to upload the dataset to a new created bucket in the localstack server (make sure to replace /path/to/PS... with the directory you downloaded the data set to):
+      ```bash
+      export ENDPOINT="http://127.0.0.1:4566"
+      export BUCKET="demo"
+      export OBJECT_KEY="PS_20174392719_1491204439457_log.csv"
+      export FILEPATH="/path/to/PS_20174392719_1491204439457_log.csv"
+      export REGION=theshire
+      aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY}
+      aws configure set region ${REGION}
+      aws --endpoint-url=${ENDPOINT} s3api create-bucket --bucket ${BUCKET} --region ${REGION} --create-bucket-configuration LocationConstraint=${REGION}
+      aws --endpoint-url=${ENDPOINT} s3api put-object --bucket ${BUCKET} --key ${OBJECT_KEY} --body ${FILEPATH}
+      ```
+
+In this step you are performing the role of the data owner, registering his data in the data catalog and registering the credentials for accessing the data in the credential manager.
+
+We now explain how to register a dataset in the Katalog data catalog.
+
+Begin by registering the credentials required for accessing the dataset as a kubernetes secret. Replace the values for `access_key` and `secret_key` with the values from the object storage service that you used and run:
+
+```bash
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+    name: paysim-csv
+type: Opaque
+stringData:
+    access_key: "${ACCESS_KEY}"
+    secret_key: "${SECRET_KEY}"
+EOF
+```
+
+Next, register the data asset itself in the data catalog.
+We use port-forwarding to send asset creation requests to the Katalog connector.
+
+```bash
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
+metadata:
+  name: paysim-csv
+spec:
+  secretRef: 
+    name: paysim-csv
+  details:
+    dataFormat: csv
+    connection:
+      name: s3
+      s3:
+        endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+        bucket: "demo"
+        object_key: "PS_20174392719_1491204439457_log.csv"
+  metadata:
+    name: Synthetic Financial Datasets For Fraud Detection
+    geography: theshire 
+    tags:
+      finance: true
+EOF
+```
+
+### Define data access policy
```

Comment:
This section is not needed because the default Fybrik deployment assumes "allow-by-default" policy.